### PR TITLE
feat: unify v9 babel preset in all packages

### PR DESCRIPTION
**Pull request by Hotell**
## Previous Behavior

react-provider uses different babel setup for griffel processing

## New Behavior

react-provider uses same babel setup for griffel.

## Related Issue(s)

- Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## Perf Analysis (`@fluentui/react-northstar`)
:warning: 1 potential perf regressions detected

### Potential regressions comparing to master

Scenario | Current PR Ticks | Baseline Ticks | Ratio | Regression Analysis
:--- | ---:| ---:| ---: | ---: 
AccordionMinimalPerf.default | [83](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/AccordionMinimalPerf.default.html) | [78](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/AccordionMinimalPerf.default_base.html) | 1.06:1 | [analysis](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/AccordionMinimalPerf.default.regression.txt)
<details><summary>Perf tests with no regressions</summary>

Scenario | Current PR Ticks | Baseline Ticks | Ratio
:--- | ---:| ---:| ---:
AvatarMinimalPerf.default | [122](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/AvatarMinimalPerf.default.html) | [108](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/AvatarMinimalPerf.default_base.html) | 1.13:1
InputMinimalPerf.default | [562](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/InputMinimalPerf.default.html) | [505](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/InputMinimalPerf.default_base.html) | 1.11:1
BoxMinimalPerf.default | [209](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/BoxMinimalPerf.default.html) | [190](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/BoxMinimalPerf.default_base.html) | 1.1:1
ListWith60ListItems.default | [383](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ListWith60ListItems.default.html) | [347](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ListWith60ListItems.default_base.html) | 1.1:1
AttachmentMinimalPerf.default | [83](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/AttachmentMinimalPerf.default.html) | [77](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/AttachmentMinimalPerf.default_base.html) | 1.08:1
AttachmentSlotsPerf.default | [696](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/AttachmentSlotsPerf.default.html) | [645](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/AttachmentSlotsPerf.default_base.html) | 1.08:1
IconMinimalPerf.default | [390](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/IconMinimalPerf.default.html) | [360](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/IconMinimalPerf.default_base.html) | 1.08:1
VideoMinimalPerf.default | [449](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/VideoMinimalPerf.default.html) | [417](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/VideoMinimalPerf.default_base.html) | 1.08:1
RefMinimalPerf.default | [112](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/RefMinimalPerf.default.html) | [105](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/RefMinimalPerf.default_base.html) | 1.07:1
ButtonSlotsPerf.default | [318](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ButtonSlotsPerf.default.html) | [300](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ButtonSlotsPerf.default_base.html) | 1.06:1
ChatDuplicateMessagesPerf.default | [153](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ChatDuplicateMessagesPerf.default.html) | [145](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ChatDuplicateMessagesPerf.default_base.html) | 1.06:1
LayoutMinimalPerf.default | [204](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/LayoutMinimalPerf.default.html) | [192](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/LayoutMinimalPerf.default_base.html) | 1.06:1
RadioGroupMinimalPerf.default | [273](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/RadioGroupMinimalPerf.default.html) | [258](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/RadioGroupMinimalPerf.default_base.html) | 1.06:1
ReactionMinimalPerf.default | [213](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ReactionMinimalPerf.default.html) | [202](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ReactionMinimalPerf.default_base.html) | 1.05:1
AnimationMinimalPerf.default | [311](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/AnimationMinimalPerf.default.html) | [300](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/AnimationMinimalPerf.default_base.html) | 1.04:1
ButtonOverridesMissPerf.default | [659](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ButtonOverridesMissPerf.default.html) | [633](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ButtonOverridesMissPerf.default_base.html) | 1.04:1
FlexMinimalPerf.default | [156](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/FlexMinimalPerf.default.html) | [150](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/FlexMinimalPerf.default_base.html) | 1.04:1
ChatMinimalPerf.default | [447](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ChatMinimalPerf.default.html) | [432](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ChatMinimalPerf.default_base.html) | 1.03:1
FormMinimalPerf.default | [228](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/FormMinimalPerf.default.html) | [221](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/FormMinimalPerf.default_base.html) | 1.03:1
HeaderSlotsPerf.default | [475](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/HeaderSlotsPerf.default.html) | [461](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/HeaderSlotsPerf.default_base.html) | 1.03:1
LabelMinimalPerf.default | [221](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/LabelMinimalPerf.default.html) | [215](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/LabelMinimalPerf.default_base.html) | 1.03:1
MenuMinimalPerf.default | [515](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/MenuMinimalPerf.default.html) | [499](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/MenuMinimalPerf.default_base.html) | 1.03:1
SegmentMinimalPerf.default | [209](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/SegmentMinimalPerf.default.html) | [203](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/SegmentMinimalPerf.default_base.html) | 1.03:1
SplitButtonMinimalPerf.default | [2291](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/SplitButtonMinimalPerf.default.html) | [2232](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/SplitButtonMinimalPerf.default_base.html) | 1.03:1
TextMinimalPerf.default | [203](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/TextMinimalPerf.default.html) | [198](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/TextMinimalPerf.default_base.html) | 1.03:1
DividerMinimalPerf.default | [200](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/DividerMinimalPerf.default.html) | [196](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/DividerMinimalPerf.default_base.html) | 1.02:1
ListNestedPerf.default | [330](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ListNestedPerf.default.html) | [325](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ListNestedPerf.default_base.html) | 1.02:1
MenuButtonMinimalPerf.default | [966](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/MenuButtonMinimalPerf.default.html) | [944](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/MenuButtonMinimalPerf.default_base.html) | 1.02:1
PopupMinimalPerf.default | [355](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/PopupMinimalPerf.default.html) | [347](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/PopupMinimalPerf.default_base.html) | 1.02:1
TextAreaMinimalPerf.default | [298](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/TextAreaMinimalPerf.default.html) | [292](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/TextAreaMinimalPerf.default_base.html) | 1.02:1
CheckboxMinimalPerf.default | [1166](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/CheckboxMinimalPerf.default.html) | [1151](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/CheckboxMinimalPerf.default_base.html) | 1.01:1
DialogMinimalPerf.default | [440](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/DialogMinimalPerf.default.html) | [437](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/DialogMinimalPerf.default_base.html) | 1.01:1
DropdownManyItemsPerf.default | [398](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/DropdownManyItemsPerf.default.html) | [393](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/DropdownManyItemsPerf.default_base.html) | 1.01:1
GridMinimalPerf.default | [198](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/GridMinimalPerf.default.html) | [196](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/GridMinimalPerf.default_base.html) | 1.01:1
ImageMinimalPerf.default | [224](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ImageMinimalPerf.default.html) | [221](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ImageMinimalPerf.default_base.html) | 1.01:1
ItemLayoutMinimalPerf.default | [706](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ItemLayoutMinimalPerf.default.html) | [697](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ItemLayoutMinimalPerf.default_base.html) | 1.01:1
ProviderMinimalPerf.default | [198](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ProviderMinimalPerf.default.html) | [196](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ProviderMinimalPerf.default_base.html) | 1.01:1
SliderMinimalPerf.default | [748](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/SliderMinimalPerf.default.html) | [743](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/SliderMinimalPerf.default_base.html) | 1.01:1
TableManyItemsPerf.default | [1116](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/TableManyItemsPerf.default.html) | [1105](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/TableManyItemsPerf.default_base.html) | 1.01:1
CustomToolbarPrototype.default | [1493](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/CustomToolbarPrototype.default.html) | [1476](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/CustomToolbarPrototype.default_base.html) | 1.01:1
ButtonMinimalPerf.default | [85](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ButtonMinimalPerf.default.html) | [85](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ButtonMinimalPerf.default_base.html) | 1:1
EmbedMinimalPerf.default | [1855](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/EmbedMinimalPerf.default.html) | [1864](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/EmbedMinimalPerf.default_base.html) | 1:1
HeaderMinimalPerf.default | [211](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/HeaderMinimalPerf.default.html) | [210](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/HeaderMinimalPerf.default_base.html) | 1:1
RosterPerf.default | [1559](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/RosterPerf.default.html) | [1564](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/RosterPerf.default_base.html) | 1:1
ProviderMergeThemesPerf.default | [644](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ProviderMergeThemesPerf.default.html) | [646](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ProviderMergeThemesPerf.default_base.html) | 1:1
TableMinimalPerf.default | [229](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/TableMinimalPerf.default.html) | [230](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/TableMinimalPerf.default_base.html) | 1:1
ToolbarMinimalPerf.default | [551](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ToolbarMinimalPerf.default.html) | [550](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ToolbarMinimalPerf.default_base.html) | 1:1
TreeMinimalPerf.default | [468](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/TreeMinimalPerf.default.html) | [466](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/TreeMinimalPerf.default_base.html) | 1:1
CardMinimalPerf.default | [304](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/CardMinimalPerf.default.html) | [306](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/CardMinimalPerf.default_base.html) | 0.99:1
DatepickerMinimalPerf.default | [3711](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/DatepickerMinimalPerf.default.html) | [3752](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/DatepickerMinimalPerf.default_base.html) | 0.99:1
DropdownMinimalPerf.default | [1420](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/DropdownMinimalPerf.default.html) | [1439](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/DropdownMinimalPerf.default_base.html) | 0.99:1
ListMinimalPerf.default | [309](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ListMinimalPerf.default.html) | [313](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ListMinimalPerf.default_base.html) | 0.99:1
PortalMinimalPerf.default | [87](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/PortalMinimalPerf.default.html) | [88](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/PortalMinimalPerf.default_base.html) | 0.99:1
SkeletonMinimalPerf.default | [199](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/SkeletonMinimalPerf.default.html) | [201](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/SkeletonMinimalPerf.default_base.html) | 0.99:1
StatusMinimalPerf.default | [387](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/StatusMinimalPerf.default.html) | [389](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/StatusMinimalPerf.default_base.html) | 0.99:1
CarouselMinimalPerf.default | [249](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/CarouselMinimalPerf.default.html) | [254](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/CarouselMinimalPerf.default_base.html) | 0.98:1
LoaderMinimalPerf.default | [194](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/LoaderMinimalPerf.default.html) | [199](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/LoaderMinimalPerf.default_base.html) | 0.97:1
TreeWith60ListItems.default | [84](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/TreeWith60ListItems.default.html) | [87](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/TreeWith60ListItems.default_base.html) | 0.97:1
ListCommonPerf.default | [385](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ListCommonPerf.default.html) | [399](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ListCommonPerf.default_base.html) | 0.96:1
AlertMinimalPerf.default | [156](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/AlertMinimalPerf.default.html) | [164](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/AlertMinimalPerf.default_base.html) | 0.95:1
TooltipMinimalPerf.default | [1242](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/TooltipMinimalPerf.default.html) | [1306](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/TooltipMinimalPerf.default_base.html) | 0.95:1
ChatWithPopoverPerf.default | [181](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ChatWithPopoverPerf.default.html) | [193](https://fluentuipr.z22.web.core.windows.net/pull/31088/perf-test-northstar/ChatWithPopoverPerf.default_base.html) | 0.94:1

</details><div id="perfComment9424"/>

**Comment by fabricteam** [comment]
## 🕵 FluentUIV0 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

File: scripts/babel/preset-v9.js [hunk 1]
```diff
@@ -56,6 +56,14 @@ const preset = (api, options) => {
             ],
           ],
         },
+        /**
+         * https://github.com/microsoft/griffel/tree/main/packages/babel-preset#importing-griffel-from-custom-packages
+         * "By default, preset handles imports from @griffel/react & @fluentui/react-components" - under the hood griffel preset wont merge rather override thus we need to provide the default modules explicitly
+         */
+        modules: [
+          { moduleSource: '@griffel/core', importName: 'makeStyles' },
```
Comment (layershifter) [inline scripts/babel/preset-v9.js:64]:
```suggestion
          { moduleSource: '@griffel/core', importName: 'makeStyles' },
          { moduleSource: '@griffel/react', importName: 'makeStyles' },
```

**Review by layershifter — APPROVED** [review]

File: scripts/babel/preset-v9.js (hunk 1 above)
Comment (Hotell) [reply scripts/babel/preset-v9.js:67]: done

**Review by layershifter — APPROVED** [review]

**Review by tudorpopams — APPROVED** [review]

**Pull request merged**